blaze-core/src/test/scala/org/http4s/blaze/util/EntityBodyWriterSpec.scala

```diff
@@ -232,34 +228,30 @@ class EntityBodyWriterSpec extends Http4sSpec {
 
     "write a deflated resource" in {
       val p = resource through deflate()
-      p.runLog.map(_.toArray) must returnValue(DumpingWriter.dump(p))
+      p.runLog.map(_.toArray) must returnValue(DumpingWriter.dump(resource through deflate()))
```

Comment:
Running a stream that is piped through a `deflate` step gives an exception with "Deflater closed", for every way I tried to do it. I checked in fs2 0.9.7 as well, and that gave the same result.

I'm not sure how this test doesn't fail on master.